packages/widget/src/index.tsx

```diff
@@ -152,6 +152,19 @@ export const Widget = forwardRef<
   { expand: () => void; collapse: () => void },
   Props
 >((props, ref) => {
+  const [windowInnerHeight, setWindowInnerHeight] = useState<number | null>(null);
+
+  useEffect(() => {
+    setWindowInnerHeight(window.innerHeight);
```

Comment:
I want to ask you to change it to `setCurrentWindowInnerHeight` or `setWindowInnerHeightValue` because it looks like you actually setting window inner height from the name

Reply:
That makes sense - done!